egs/librispeech/ASR/conformer_ctc/train.py

```diff
@@ -606,6 +606,35 @@ def run(rank, world_size, args):
     train_dl = librispeech.train_dataloaders()
     valid_dl = librispeech.valid_dataloaders()
 
+    from lhotse.dataset import find_pessimistic_batches
```

Comment:
I find that there is no `find_pessimistic_batches`
in the latest https://github.com/lhotse-speech/lhotse/blob/master/lhotse/dataset/__init__.py

Do you forget to commit it to lhotse?